src/components/network/network-panel.js

```diff
@@ -111,6 +120,40 @@ genotet.NetworkPanel.prototype.edgeListContainer = function() {
 };
 
 /**
+ * Updates the network list for panel after loading network list.
+ */
+genotet.NetworkPanel.prototype.updateFileListAfterLoading = function() {
```

Comment:
Possible TODO: this function is now very similar with expressionPanel.prototype.updateFileListAfterLoading. Consider create a shared UI class "genotet.FileList". Each panel can create instances of this UI class and set the file list by calling the UI class's methods.